recipe/gen_patch_json.py

```diff
@@ -239,6 +239,22 @@ def rename_dependency(fn, record, old_name, new_name):
             depends[dep_idx] = new_name + remainder
             instructions["packages"][fn]['depends'] = depends
 
+    def fix_libgfortran(fn, record):
+        depends = record.get("depends", ())
+        dep_idx = next(
+            (q for q, dep in enumerate(depends) if dep.split(' ')[0] == "libgfortran"),
+            None
+        )
+        if dep_idx:
+            # make sure respect minimum versions still there
+            # default to a min of 3.0.1 if none is given
+            if ">=3.0.1" in depends[dep_idx] or depends[dep_idx] == "libgfortran":
```

Comment:
There's also `libgfortran >=3.0` that needs to be addressed.

Reply:
ack yes this patch is annoying...

Reply:
Actually no. I think `>=3.0` is caught by the last else case.

Reply:
Done.